src/sass/components/Pagination.scss

```diff
@@ -26,7 +26,16 @@
   opacity: 1;
   transition: opacity 0.3s;
 
+  td {
+    opacity: 1;
+    transition: opacity 0.3s;
+  }
   &.fadeOut {
-    opacity: 0;
+    opacity: 0.5;
+    box-shadow: 0 1px 1px -2px;
```

Comment:
Bottom border of the table? I tried with and without and didn't notice anything different.
If you think it needs to be there maybe just a quick comment to tell why.